apps/rush-lib/src/cli/actions/PublishAction.ts

```diff
@@ -221,6 +222,8 @@ export class PublishAction extends BaseRushAction {
       this._prereleaseToken,
       this._addCommitDetails.value);
 
+    const storeLooseVersions: boolean = this.rushConfiguration.storeLooseVersions;
```

Comment:
In theory, rather than editing the `package.json` files four times, we could instead overhaul the version-bump mechanics to account for this new behavior. However, I imagine such a refactor would be a *lot* more error-prone. For the moment, treat this behavior as a 'beta' to see if it's practical.

Reply:
I want to do that soon anyway, so it's good that we're talking about this now.